internal/logs/aggregate/system.go

```
@@ -0,0 +1,276 @@
+	if stale && !systemStatsUpdateInFlight {
+		systemStatsUpdateInFlight = true
+		go refreshSystemStats()
+	}
+	systemStatsMutex.Unlock()
+
+	return stats
+}
+
+func refreshSystemStats() {
+	defer func() {
+		if recover() != nil {
```

golangci-lint (2.12.2)

[error] SA9003: empty branch

(staticcheck)